Common/Data/Auxiliary/FactorFileRow.cs

```diff
@@ -113,29 +113,20 @@ public static IEnumerable<FactorFileRow> Read(string permtick, string market, ou
         /// <returns>An enumerable of factor file rows</returns>
         public static List<FactorFileRow> Parse(IEnumerable<string> lines, out DateTime? factorFileMinimumDate)
         {
-            var hasInfEntry = false;
             factorFileMinimumDate = null;
 
             var rows = new List<FactorFileRow>();
 
             // parse factor file lines
             foreach (var line in lines)
             {
-                if (line.Contains("inf"))
+                if (line.Contains("inf") || line.Contains("e+"))
```

Comment:
Hmm, does the `e+` actually mean it's inf? I suppose by convention we don't serialize using scientific notation, but perhaps more correctly we could define inf as `"inf"` or anything larger than, for example, `1e6`. If we are going to say that _all_ scientific notation equates to inf, then we'll want to document this behavior somewhere since it's unexpected. For example, a value such as `1.23e+2` would be flagged unexpectedly as an inf value

Reply:
Exponential notation is treated as `inf ` is because of the loss of precision. In all cases, the represented number is a round up to a few (and sometimes wrong) decimals, E.g., 1.6e+6 when the correct factor is  1562500. I'll add that as comment.

Worth notice that there are only 6 factor files with exponential notation, 10 cases with `inf`, and 2 cases where both cases are present.